.goreleaser.yml

```diff
@@ -0,0 +1,37 @@
+project_name: attiny-controller
+
+release:
+  github:
+    owner: TheCacophonyProject
+    name: attiny-controller
+  name_template: '{{.Tag}}'
+
+builds:
+- goos:
+  - linux
+  goarch:
+  - arm
+  goarm:
+  - "7"
+  main: .
+  ldflags: -s -w -X main.version={{.Version}}
+  binary: attiny-controller
+
+nfpm:
+  vendor: The Cacophony Project
+  homepage: http://cacophony.org.nz/
+  maintainer: Menno Finlay-Smits <menno@cacophony.org.nz>
+  description: Controls the ATtiny on the Pi Hat
+  license: GPL v3.0
+  name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
+  formats:
+    - deb
+  bindir: /usr/bin
+  files:
+    "attiny-controller.service": "/etc/systemd/system/attiny-controller.service"
```

Comment:
I've started putting all the release/deployment related files in a `_release` subdirectory in the source tree. Keeps it out of the way.